### ISSUE TITLE
Add support for sourcemaps

### ISSUE THREAD

**Issue by nicolasgarnier** [ragingwind/gulp-crisper#4]
Gulp-crisper is great but it would be even more awesome if it had support for sourcemaps generation.

For instance I should be able to do this:

``` javascript
gulp.task('separateJS', function () {
  return gulp.src(['app/**/*.html'])
    .pipe($.sourcemaps.init())
    .pipe($.crisper()) // Extract JS from .html files
    .pipe($.sourcemaps.write('.'))
    .pipe(gulp.dest('dist/'))
});
```

Some docs for you: https://www.npmjs.com/package/gulp-sourcemaps#plugin-developers-only-how-to-add-source-map-support-to-plugins

**Comment by nicolasgarnier** [comment]
FYI I filed https://github.com/PolymerLabs/crisper/issues/14 asking to add sourcemaps generation support in the Crisper binary. Seems like you need it first.

**Comment by ragingwind** [comment]
Yes, I need supporting of vulcanize bin and then I can pass a source map file stream out
